src/bloqade/task/braket_simulator.py

```diff
@@ -19,17 +24,48 @@
 @dataclass
 @Serializer.register
 class BraketEmulatorTask(LocalTask):
+    """
+    Represents a quantum task that runs on the AWS Braket LocalSimulator.
+
+    Attributes:
+        task_ir (BraketTaskSpecification): The task specification for the Braket simulator.
+        metadata (Dict[str, ParamType]): Metadata related to the task.
+        task_result_ir (Optional[QuEraTaskResults]): The results of the task, if available.
+    """
+
     task_ir: BraketTaskSpecification
     metadata: Dict[str, ParamType]
     task_result_ir: Optional[QuEraTaskResults] = None
 
     def _geometry(self) -> Geometry:
+        """
```

Comment:
This is a private function, and I'm doing duplicated reviews on the same issue. Please work on one of your PRs first before requesting multiple reviews thank you.